<launch>
    <arg name="simulation" default="$(optenv SIM false)" />
    <arg name="challenge4" default="$(optenv CHALLENGE4 true)"/>
    <arg name="docker" default="$(optenv DOCKER_RUN false)"/>

    <param name="simulation" type="bool" value="$(arg simulation)" />
    <param name="challenge4" type="bool" value="$(arg challenge4)" />

    <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time: %r}] [${node} ${function} ${line}]: ${message}" />
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find drone)/config/rosconsole.conf" />

    <rosparam command="load" file="$(find drone)/config/config.yaml" />

    <!-- Our node to control the drone -->
    <node pkg="drone" type="main.py" name="rob498_drone_07" required="true" output="screen" >
    </node>

    <!-- Real Drone-->
    <include file="$(find drone)/launch/bridge_mavros.launch" unless="$(arg simulation)">

    </include>



</launch>
